rsocket-core/build.gradle

```diff
@@ -30,7 +30,9 @@ dependencies {
     implementation 'org.slf4j:slf4j-api'
 
     compileOnly 'com.google.code.findbugs:jsr305'
-
+    
+    implementation 'org.agrona:agrona:0.9.33'
```

Comment:
We probably shouldn't pull this in - we can just add the BufferUtil.allocateDirectAligned method into a util class in RSocket so we don't need a new dependency.

Reply:
Alright

Reply:
fixed